test/index.test.ts

```diff
@@ -131,6 +131,17 @@ describe(degit, { timeout }, () => {
 		});
 	});
 
+	describe.sequential('Codeberg', () => {
+		it.each([
```

Comment:
Hi, @nake89!

I think we need a Codeberg repo for testing. Are you going to create one on your side or do you want me to create one on my side?

Reply:
You can create one.

I might clone your repo to my codeberg later and change the test to match my repo url. But for now it is fine to just use yours.

Reply:
Got it, thanks!